### DISCUSSION TITLE
Add ability to specify bucket name when adding a new s3 connection

### DISCUSSION THREAD

**Discussion by hfabre** [iterate-ch/cyberduck#16463] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
I have some s3 keys which don't grant access to `s3:ListAllMyBuckets`. So unless i missed an option somewhere, i can't use cyberduck to access the bucket i would like to.

**Describe the solution you'd like**
I would like to have a new optional field in the "new connection" form for s3 to specify the bucket name so cyberduck does not need to list all buckets.

**Describe alternatives you've considered**
I don't think there is a lot of alternatives

**Additional context**
Here is the form i'm talking about
<img width="545" alt="image" src="https://github.com/user-attachments/assets/ac903fb4-8a70-4f0a-8335-5613bee30406">

Note: I may have missed an existing option in this case sorry for the inconvenience. Anyway thank you for this awesome software !

**Comment by dkocher** [comment, marked as answer]
Duplicate for #12891. Instead choose _Bookmarks → New Bookmark_ where you have the option to set the bucket name in the _Path_ field. Alternatively set the bucket name as part of the _Server_ using the notation `bucketname.s3.amazonaws.com`.

**Reply by hfabre** [reply]
Got it thanks for the fast reply !
